tests/data/group_system/group_software/group_gnome/dconf_test_functions.sh

```diff
@@ -4,6 +4,11 @@ clean_dconf_settings(){
 	rm -rf /etc/dconf/db/*
 }
 
```

Comment:
Could you please add a function `install_dconf_and_gdm_if_needed` and call it from tests? That way, copy-pasting will be reduced and tests will be more readable as well.

Reply:
Done